src/interfaces/business-document-interface.ts

```diff
@@ -0,0 +1,23 @@
+// This comes back from the business API documents call for a business - see LegalService.fetchDocumentInfo
+export interface DocumentInfoContentIF {
+  includedInResolution: boolean
+  includedInResolutionDate?: string
+  key: string,
+  name: string,
+  uploaded: string
+}
+
+export interface DocumentInfoIF {
+  certifiedMemorandum?: DocumentInfoContentIF
+  certifiedRules?: DocumentInfoContentIF
+}
```

Comment:
This is all very coop-centric. I don't know what the future looks like so I'm OK with not making this super-flexible right now, but how about naming these interfaces coop-like? Thoughts?

(On the other hand, all the optional properties makes this extensible/reusable already. 🤷‍♂️ )

Reply:
Yes, but it can be expanded to other documents (already is for the documents below).

DocumentInfo is just the extra tidbit that matches the Document KV

Reply:
Sure. Leave it as is then.